### ISSUE TITLE
Revise CoreTrustSeal Application

### ISSUE THREAD

**Issue by mikkohei13** [luomus/meta#60]
Revision of the application needed.

v2 to be edited: https://helsinkifi.sharepoint.com/:w:/s/BDIUnit/Ea5A22Q4EmRNvTTJfnx7x1cBKbOmmm5W3PRfnxJgOb1bcA?e=ZIUm9W

**Comment by wkmor1** [comment]
Has the last CTS submission been returned to us? Is there a deadline?

**Comment by mikkohei13** [comment]
Yes, the reviewers want a revision for 7 out of 20 sections, mostly more details. Deadline January 10th 2025.

**Comment by esko-piirainen** [comment]
One more revision needed. Other than that approved.

**Comment by esko-piirainen** [comment]
I wrote additions to the chapter that still needed revisions.

**Comment by esko-piirainen** [comment]
Revised document with final unapproved chapter has been submitted
